ui-admin/src/forms/designer/split/FullQuestionDesigner.tsx

```diff
@@ -0,0 +1,159 @@
+      <span className="text-danger fw-semibold ms-1">*</span>
+      <div className="d-flex align-items-center">
+        <div className="w-100">
+          <TextInput
+            className={'mb-2'}
+            required={true}
+            value={question.name}
+            disabled={isStableIdLocked}
+            onChange={name => {
+              onChange({
+                ...question,
+                name
+              })
+            }}
+          />
+        </div>
+        <IconButton
+          className="mb-2"
+          icon={isStableIdLocked ? faLock : faUnlock}
+          aria-label={isStableIdLocked ? 'Edit stable ID' : 'Lock stable ID'}
+          onClick={() => setIsStableIdLocked(!isStableIdLocked)}
+        />
+        <IconButton
+          className={classNames('mb-2', { 'text-danger': question.isRequired })}
+          icon={faAsterisk}
+          aria-label={'Toggle required'}
+          onClick={() => {
+            onChange({
+              ...question,
+              isRequired: !question.isRequired
+            })
+          }}
+        />
+      </div>
+      <QuestionTypeSelector
+        key={question.name}
+        // @ts-ignore
+        questionType={question.type}
+        onChange={newType => {
+          onChange({
+            ...baseQuestions[newType as QuestionType],
+            ...question,
+            // @ts-ignore
+            type: newType
+          })
+        }}
+      />
+
+      {isTemplated && (
+        <>
+          <p className="fs-4 mb-0">Templated question</p>
+          <p>
+              This question uses <span className="fw-bold">{question.questionTemplateName}</span> as a template.
+              Question settings entered here override settings from the template.
+          </p>
+        </>
+      )}
+
+      <BaseFields
+        showIsRequired={false}
+        disabled={readOnly}
+        currentLanguage={currentLanguage}
+        supportedLanguages={supportedLanguages}
+        question={question}
+        onChange={onChange}
+      />
+
+      {!isTemplated && (
+        <>
+          {
+            (question.type === 'checkbox' || question.type === 'dropdown' || question.type === 'radiogroup') && (
+              <>
+                <ChoicesList
+                  question={question}
+                  isNewQuestion={isNewQuestion}
+                  currentLanguage={currentLanguage}
+                  supportedLanguages={supportedLanguages}
+                  readOnly={readOnly}
+                  onChange={onChange}
+                />
+              </>
+            )
+          }
+          {
+            question.type === 'text' && (
+              <TextFields
+                disabled={readOnly}
+                question={question}
+                onChange={onChange}
+              />
+            )
+          }
+          {
+            question.type === 'html' && <Textarea
+              disabled={readOnly}
```

Comment:
I think this needs a bit of padding, it current touches the area beneath it: 
```suggestion
            question.type === 'html' && <div className="mb-2"><Textarea
              disabled={readOnly}
```